### DISCUSSION TITLE
I got an error from undervolting(0.87)

### DISCUSSION THREAD

**Discussion by envyska** [seerge/g-helper#643] [Q&A] [answered]
![bug](https://github.com/seerge/g-helper/assets/136965993/26ebf534-2f6d-45ce-9721-e4798fbd1d7c)

The type initiallizer for ~~ exception. 

maybe user adress problem?

**Comment by seerge** [comment, marked as answer]
@envyska You need to unpack whole zip (including 2 libraries) into same folder.

**Reply by envyska** [reply]
ty for help :)
